x-pack/plugins/cases/public/components/custom_fields/index.test.tsx

```diff
@@ -97,13 +97,11 @@ describe('CustomFields', () => {
         required: false,
       });
     }
+
     const customFields = [...customFieldsConfigurationMock, ...generatedMockCustomFields];
 
     appMockRender.render(<CustomFields {...{ ...props, customFields }} />);
 
-    await userEvent.click(await screen.findByTestId('add-custom-field'));
-
     expect(await screen.findByText(i18n.MAX_CUSTOM_FIELD_LIMIT(MAX_CUSTOM_FIELDS_PER_CASE)));
-    expect(await screen.findByTestId('add-custom-field')).toHaveAttribute('disabled');
```

Comment:
nit: we can update it to check there is no `add-custom-field` button visible. like expect(screen.queryByTestId('add-custom-field')).not.toBeInTheDocument();

Reply:
done, thanks!